tests/test_bias.py

```diff
@@ -0,0 +1,168 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import os
+import numpy as np
+import unittest
+
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+import lsst.ip.isr as ipIsr
+import lsst.meas.algorithms as measAlg
+import lsst.utils.tests
+
+from lsst.pipe.tasks.repair import RepairTask
+
+
+# TODO: DM-26396
+#       Update these tests to validate calibration construction.
+class BiasTestCases(lsst.utils.tests.TestCase):
+
+    def setUpClass(self):
+        """Setup butler, and generate an ISR processed exposure.
+
+        Notes
+        -----
+        DMTN-101 4.1:
+
+        Process an independent bias frame through the ISR including
+        overscan correction and bias subtraction
+
+        """
+        repoDir = os.path.join("..", "DATA")
+        calibDir = os.path.join("..", "DATA", "calibs")
+        butler = dafPersist.Butler(repoDir, calibRoot=calibDir)
+
+        self.config = ipIsr.IsrTaskConfig()
+        self.config.doSaturation = True
+        self.config.doSuspect = True
+        self.config.doSetBadRegions = True
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doDefect = False
+        self.config.doSaturationInterpolation = False
+        self.config.doDark = False
+        self.config.doStrayLight = False
+        self.config.doFlat = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
+        self.dataRef = butler.dataRef('raw', dataId={'detector': 0, 'expId': 2020012800007})
+        results = self.isrTask.runDataRef(self.dataRef)
+        self.exposure = results.outputExposure
+
+    def test_independentFrameLevel(self):
+        """Test image mean.
+
+        Notes
+        -----
+        DMTN-101 4.2
+
+        Confirm that the mean of the result is 0 to within statistical
+        error
+        """
+        mean = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEAN).getValue()
+        self.assertLess(np.abs(mean), 1, msg=f"Test 4.2: {mean}")
```

Comment:
I guess if the tech note says 1 is a magic value then it's maybe legit to hard code it here, but I do wonder. It's using an unclipped mean though, so I can imagine a very bright column or few throwing that out, for example. On the other hand, we're also testing defect finding and application, so maybe that's exactly what we want... Something to think on though.

Reply:
(that's just one of many failure modes that spring to mind though)

Reply:
OK, fair enough. I'm not sure that totally addresses the hard-coding of this value, but I guess it gives us a chance to revisit all the tests and their threshold values, so yeah, let's resolve for now 👍